lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -735,6 +735,19 @@ OpFoldResult AtenDimOp::fold(ArrayRef<Attribute> operands) {
   return nullptr;
 }
 
+//===----------------------------------------------------------------------===//
+// AtenIsFloatingPointOp
+//===----------------------------------------------------------------------===//
+
+OpFoldResult AtenIsFloatingPointOp::fold(ArrayRef<Attribute> operands) {
+  if (auto tensorType = getOperand().getType().dyn_cast<BaseTensorType>()) {
+    if (typeid(tensorType).name() == "f")
+      return BoolAttr::get(BoolType::get(getContext(), 64),
+		           tensorType.getSizes().size());
```

Comment:
The arguments to `BoolAttr::get` are `MLIRContext *, bool` (https://mlir.llvm.org/doxygen/classmlir_1_1BoolAttr.html#adb387d09645c6a87af1e6dec16791bdc). Currently, `BoolType, int` are being passed